spec/reference/protocol.mdx

```diff
@@ -224,6 +224,27 @@ If these are handled purely through events, the client will maintain the pressed
 For this, it's critical not to swap order of up/down events.
 To recognize button combinations, it's important not to swap order of events from the two services.
 
+#### Reliable actions
+
+Devices can support reliable actions, which use the same encoding as events, but are commands, not reports.
+The `service_command` field is subdivided as follows:
+
+| Bits | Function 
+|------|----------------------
+| 15   | Always set for reliable actions
+| 14:8 | Sequential counter
+| 7:0  | Action code
+
+Only action codes in range `0x00-0xff` are supported.
+Just as for events, the sequential counter is per-device (but separate from the event sequential counter).
```

Comment:
This doesn't work, since the client doesn't know the counter on the server. If client did know that, there could be collisions.